exports/Taskfile.dist.yaml

```diff
@@ -15,8 +15,10 @@ includes:
     optional: true # Optional as we may not have any local taskfiles
     aliases: [l]
 
-# Load our project specific variables from .env.taskit
-dotenv: [.env.taskit]
+# Load our project specific variables from .env.taskit + .env.taskit-secrets
+dotenv:
+  - .env.taskit
+  - .env.taskit-secrets
```

Comment:
Where is taskit/snaplet typically run? Can we pass these in a way other than .env files?

Reply:
I'm wondering for 12-factor purposes, but it looks like taskfile is the one handling the `.env` import rather than us, so any 12-factor concerns are left on their plate, which I'm cool with

Reply:
We're typically running snaplet tasks in our client infra project where these files are either present or in the case of the `taskit-secrets` file it can be created easily if needed. If these files don't exist, task / taskit works as normal. They're purely optional and supplementary which works nicely if we are using taskit in a project that doesn't use the `snaplet:*` tasks.